twindb_backup/destination/local.py

```diff
@@ -45,51 +53,33 @@ def list_files(self, prefix, recursive=False):
             ls_cmd = ["ls", "%s*" % prefix]
 
         cmd = ls_cmd
-        log.debug('Running %s', ' '.join(cmd))
-        proc = Popen(cmd, stdout=PIPE, stderr=PIPE)
-        cout, cerr = proc.communicate()
 
-        return sorted(cout.split())
+        with run_command(cmd) as cout:
+            return sorted(cout.split())
 
-    def find_files(self, prefix):
+    def find_files(self, prefix, run_type):
```

Comment:
It doesn't because run_type is an argument of the parent method. Actually, it complains when I remove it.